api/tests/conftest.py

```diff
@@ -0,0 +1,24 @@
+import pytest
+from flask import Flask
+
+def create_app(test_config=None):
+    app = Flask(__name__, static_folder='./build', static_url_path='/')
+
+    @app.route('/api/status')
```

Comment:
Don't you want to reference the original reference instead of copy the code here?